challenges/funding.md

```diff
@@ -0,0 +1,108 @@
+# Funding
+
+Funding RSE can be challenging as we are often split between different forms of funding
+
+## Glossary
+
+- **Directly Incurred (DI)**: Costs that are directly attributable to a specific
```

Comment:
The UCL research services definitions may be helpful here:
* Directly Incurred costs are project-specific (i.e., they arise as a direct consequence of the project taking place), actual, and must be auditable at project level (e.g., supported by supplier invoices).
* Directly Allocated costs are not project-specific (i.e., they are incurred whether or not the project takes place), and are estimated at the project level e.g. investigator time, technician time (where not directly incurred), and estates costs.
* Indirect costs represent the costs of central and distributed services shared by other activities that are not project-specific e.g. Library services, Finance, Human Resources, and IT.